pom.xml

```diff
@@ -3,12 +3,12 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.springframework.samples</groupId>
   <artifactId>spring-petclinic</artifactId>
-  <version>2.7.0-SNAPSHOT</version>
+  <version>2.7.3-SNAPSHOT</version>
```

Comment:
2.7.3 is released, right? So no need for snapshots.

Reply:
Fixed.